samples/bluetooth/peripheral_hids/README.rst

```diff
@@ -15,7 +15,7 @@ In the default configuration the sample uses passkey authentication (displays a
 code on the peripheral and requires that to be entered on the host during
 pairing) and requires an authenticated link to access the GATT characteristics.
 To disable authentication and just use encrypted channels instead, build the
-sample with `CONFIG_SAMPLE_BT_USE_AUTHENTICATION=n`.
+sample with :kconfig:option:`CONFIG_SAMPLE_BT_USE_AUTHENTICATION` ``=n``.
```

Comment:
Does this render OK?

Reply:
probably not ideal (see ex. [CONFIG_SAMPLE_BT_USE_AUTHENTICATION](https://docs.zephyrproject.org/latest/connectivity/bluetooth/bluetooth-arch.html#build-types) , no, but I messed up anyway: CONFIG_SAMPLE_BT_USE_AUTHENTICATION is local to this sample so there is no point in using the Sphinx role. I will just convert the single backticks to double backticks

Reply:
So `:kconfig:option:` should not be used for sample defined Kconfig values?

Reply:
It *can* be used, it just doesn't really make a difference vs. using double backticks. In theory I guess we could check if those "local" Kconfig options exist, but that sounds overkill.